sdk/python/packages/flet/src/flet/testing/flet_test_app.py

```diff
@@ -20,7 +20,7 @@
 DEFAULT_SCREENSHOTS_PIXEL_RATIO = "2.0"
 DEFAULT_SIMILARITY_THRESHOLD = "99.0"
 
-USE_FVM = get_bool_env_var("FLET_TEST_USE_FVM")
+FLET_TEST_DISABLE_FVM = get_bool_env_var("FLET_TEST_DISABLE_FVM")
 """Whether to use FVM to run the Flutter app."""
```

Comment:
Docstring not up to date. Should be `"Whether not ..."`, since you inverted the flag.

I even think we can remove this var from the module, and directly write the below on line 116, since README already has definitions:
```py
if get_bool_env_var("FLET_TEST_DISABLE_FVM"):
```